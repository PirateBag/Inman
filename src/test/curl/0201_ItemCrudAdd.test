Add all possible items in a single batch.

===  JSON  ===
{ "updatedRows" : [
  { "summaryId" : "W-001", "description" : "36 In Red Wagon",    "unitCost" : "0.0", "sourcing" : "MAN", "leadTime" :  "2", "crudAction" : "INSERT", "quantityOnHand" : "5", "minimumOrderQuantity" : "1" },
  { "summaryId" : "W-002", "description" : "Red Wagon Body",     "unitCost" : "0.0", "sourcing" : "MAN", "leadTime" : "10", "crudAction" : "INSERT", "minimumOrderQuantity" : "1" },
  { "summaryId" : "W-003", "description" : "Front Wheel Asmbly", "unitCost" : "0.0", "sourcing" : "MAN", "leadTime" : "1", "crudAction" : "INSERT", "minimumOrderQuantity" : "1" },
  { "summaryId" : "W-004", "description" : "Rear Wheel Asmbly",  "unitCost" : "0.0", "sourcing" : "MAN", "leadTime" : "1", "crudAction" : "INSERT", "minimumOrderQuantity" : "1" },
  { "summaryId" : "W-005", "description" : "Axle",               "unitCost" : "0.0", "sourcing" : "MAN",  "leadTime" : "4","crudAction" : "INSERT", "minimumOrderQuantity" : "1" },
  { "summaryId" : "W-006", "description" : "Decal",              "unitCost" : "0.25", "sourcing" : "PUR", "leadTime" : "5", "crudAction" : "INSERT", "minimumOrderQuantity" : "1" },
  { "summaryId" : "W-007", "description" : "Washer",             "unitCost" : ".21", "sourcing" : "PUR", "leadTime" : "5", "crudAction" : "INSERT", "minimumOrderQuantity" : "1000" },
  { "summaryId" : "W-008", "description" : "Threaded bolt",      "unitCost" : "0.6", "sourcing" : "PUR", "leadTime" : "5", "crudAction" : "INSERT", "minimumOrderQuantity" : "500" },
  { "summaryId" : "W-009", "description" : "Nut",                "unitCost" : "0.17","sourcing" : "PUR", "leadTime" : "6", "crudAction" : "INSERT", "minimumOrderQuantity" : "500" },
  { "summaryId" : "W-010", "description" : "8 In Wheel",         "unitCost" : "0.17","sourcing" : "PUR", "leadTime" : "10", "crudAction" : "INSERT", "minimumOrderQuantity" : "120" },
  { "summaryId" : "W-011", "description" : "Front Wheel Bracket","unitCost" : "0.95","sourcing" : "PUR", "leadTime" : "2", "crudAction" : "INSERT", "minimumOrderQuantity" : "50" },
  { "summaryId" : "W-012", "description" : "Yoke",               "unitCost" : "0.17","sourcing" : "PUR", "leadTime" : "10", "crudAction" : "INSERT", "minimumOrderQuantity" : "50" },
  { "summaryId" : "W-013", "description" : "Wagon Body",         "unitCost" : "2.25","sourcing" : "PUR", "leadTime" : "20", "crudAction" : "INSERT", "minimumOrderQuantity" : "100" },
  { "summaryId" : "W-014", "description" : "Red Paint",          "unitCost" : "20.0", "sourcing" : "PUR", "leadTime" : "5", "crudAction" : "INSERT", "minimumOrderQuantity" : "4" },
  { "summaryId" : "W-015", "description" : "Rear Wheel bracket", "unitCost" : "1.0",  "sourcing" : "PUR", "leadTime" : "4","crudAction" : "INSERT", "minimumOrderQuantity" : "50" },
  { "summaryId" : "W-016", "description" : "Axle Bracket Brace", "unitCost" : "0.75", "sourcing" : "PUR", "leadTime" : "6", "crudAction" : "INSERT", "minimumOrderQuantity" : "50" },
  { "summaryId" : "W-017", "description" : "Rod Stock",          "unitCost" : "0.70","sourcing" : "PUR", "leadTime" : "5", "crudAction" : "INSERT", "minimumOrderQuantity" : "1000" },
  { "summaryId" : "W-018", "description" : "Carriage Bolt",      "unitCost" : "0.17", "sourcing" : "PUR", "leadTime" : "3", "crudAction" : "INSERT", "minimumOrderQuantity" : "500" }
] }
