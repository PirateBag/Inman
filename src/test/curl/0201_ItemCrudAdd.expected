{
  "responseType" : "MULTILINE",
  "errors" : [ ],
  "data" : [ {
    "id" : 1,
    "crudAction" : "NONE",
    "summaryId" : "W-001",
    "description" : "36 In Red Wagon",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 2,
    "quantityOnHand" : 5.0,
    "minimumOrderQuantity" : 1.0
  }, {
    "id" : 2,
    "crudAction" : "NONE",
    "summaryId" : "W-002",
    "description" : "Red Wagon Body",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 10,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1.0
  }, {
    "id" : 3,
    "crudAction" : "NONE",
    "summaryId" : "W-003",
    "description" : "Front Wheel Asmbly",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 1,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1.0
  }, {
    "id" : 4,
    "crudAction" : "NONE",
    "summaryId" : "W-004",
    "description" : "Rear Wheel Asmbly",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 1,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1.0
  }, {
    "id" : 5,
    "crudAction" : "NONE",
    "summaryId" : "W-005",
    "description" : "Axle",
    "unitCost" : 0.0,
    "sourcing" : "MAN",
    "maxDepth" : 0,
    "leadTime" : 4,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1.0
  }, {
    "id" : 6,
    "crudAction" : "NONE",
    "summaryId" : "W-006",
    "description" : "Decal",
    "unitCost" : 0.25,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1.0
  }, {
    "id" : 7,
    "crudAction" : "NONE",
    "summaryId" : "W-007",
    "description" : "Washer",
    "unitCost" : 0.21,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1000.0
  }, {
    "id" : 8,
    "crudAction" : "NONE",
    "summaryId" : "W-008",
    "description" : "Threaded bolt",
    "unitCost" : 0.6,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 500.0
  }, {
    "id" : 9,
    "crudAction" : "NONE",
    "summaryId" : "W-009",
    "description" : "Nut",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 6,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 500.0
  }, {
    "id" : 10,
    "crudAction" : "NONE",
    "summaryId" : "W-010",
    "description" : "8 In Wheel",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 10,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 120.0
  }, {
    "id" : 11,
    "crudAction" : "NONE",
    "summaryId" : "W-011",
    "description" : "Front Wheel Bracket",
    "unitCost" : 0.95,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 2,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 50.0
  }, {
    "id" : 12,
    "crudAction" : "NONE",
    "summaryId" : "W-012",
    "description" : "Yoke",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 10,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 50.0
  }, {
    "id" : 13,
    "crudAction" : "NONE",
    "summaryId" : "W-013",
    "description" : "Wagon Body",
    "unitCost" : 2.25,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 20,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 100.0
  }, {
    "id" : 14,
    "crudAction" : "NONE",
    "summaryId" : "W-014",
    "description" : "Red Paint",
    "unitCost" : 20.0,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 4.0
  }, {
    "id" : 15,
    "crudAction" : "NONE",
    "summaryId" : "W-015",
    "description" : "Rear Wheel bracket",
    "unitCost" : 1.0,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 4,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 50.0
  }, {
    "id" : 16,
    "crudAction" : "NONE",
    "summaryId" : "W-016",
    "description" : "Axle Bracket Brace",
    "unitCost" : 0.75,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 6,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 50.0
  }, {
    "id" : 17,
    "crudAction" : "NONE",
    "summaryId" : "W-017",
    "description" : "Rod Stock",
    "unitCost" : 0.7,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 5,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 1000.0
  }, {
    "id" : 18,
    "crudAction" : "NONE",
    "summaryId" : "W-018",
    "description" : "Carriage Bolt",
    "unitCost" : 0.17,
    "sourcing" : "PUR",
    "maxDepth" : 0,
    "leadTime" : 3,
    "quantityOnHand" : 0.0,
    "minimumOrderQuantity" : 500.0
  } ]
}